book/src/msrv.md

```diff
@@ -0,0 +1,19 @@
+# MSRV
+
+The MSRV (Minimum Supported Rust Version) is the oldest version of Rust that
+this crate is guaranteed to work with.
+
+The MSRV(s) for `flutter_rust_bridge` are as follows:
+
+| Channel | MSRV |
+| ------- | ---- |
+| stable  | <!-- stable msrv --> `1.64+` |
+| nightly | <!-- nightly msrv --> `nightly-2022-07-15+` |
```

Comment:
nit: It seems that we are repeating ourself for 3 times: 

1. the one in ci (we test our code with MSRV version)
2. the one in readme.md (the badge)
3. the doc here

is it possible we somehow add a very quick sanity check for them, to ensure that they are always in sync? e.g. some quick python/shell/whatever script to extract MSRV from these 3 places, and check they are equal. Then we can add them to CI, so nobody will accidencially make MSRV not-in-sync

---

EDIT: Oh I see, they are already auto generated from shell. Then what about this:

1. refactor the shell script into "determine MSRV automatically" and "given MSRV, write them into these few places". because in the future someone may want to manually write MSRV.
2. in these few places, maybe we can add comments saying e.g. `<!-- MSRV IS AUTOMATICALLY WRITTEN BY xxxxxxxx.sh, PLEASE DO NOT MODIFY BY HAND, BUT USE THAT SCRIPT-->`

Reply:
I had intended to do this, ill get it fixed :)